src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { isMarried, partnerName, name, age, sex } = person;
+
+  const isMarrider = isMarried ? (
+    <p className="Person__partner">
+      {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+    </p>
+  ) : (
+    <p className="Person__partner">I am not married</p>
+  );
```

Comment:
There is a typo in the variable name 'isMarrider'. It should be 'isMarried' to reflect its purpose more accurately.